Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/CallFeedback.yaml

```
@@ -0,0 +1,35 @@
+
+    findByCallId:
+      kvFunction: findOneWithKV
+      where: callId
+  
+  sqlType:
+    id: character varying (255)
+    callId: character varying (255)
+    entityId: character varying (255)
+    optionIds: "text[]"
+    createdAt: timestamp with time zone
+    
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)